### PR TITLE
EG-4409 Added installation instructions

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 1]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
 
 ## Installation
 
-This is an extension of Corda OS 4.6. If you have this version of Corda, and want to set up and run a Business Network, you can make use of the extension flows.
+This is an extension for Corda OS 4.6. If you have this version of Corda, you can access the required `.jar` files here:
+
+* BNE contracts https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-contracts/1.0/business-networks-contracts-1.0.jar
+* BNE workflows https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-workflows/1.0/business-networks-workflows-1.0.jar
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:54]:
```suggestion
* BNE workflows: [https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-workflows/1.0/business-networks-workflows-1.0.jar](https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-workflows/1.0/business-networks-workflows-1.0.jar).
```

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 2]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
...
+* Sample CorDapp contracts https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-contracts/1.0/business-networks-demo-contracts-1.0.jar
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:55]:
```suggestion
* Sample CorDapp contracts: [https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-contracts/1.0/business-networks-demo-contracts-1.0.jar](https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-contracts/1.0/business-networks-demo-contracts-1.0.jar).
```

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 3]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
...
+* Sample CorDapp workflows https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-workflows/1.0/business-networks-demo-workflows-1.0.jar
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:56]:
```suggestion
* Sample CorDapp workflows: [https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-workflows/1.0/business-networks-demo-workflows-1.0.jar](https://software.r3.com/artifactory/webapp/#/artifacts/browse/tree/General/corda-releases/net/corda/bn/business-networks-demo-workflows/1.0/business-networks-demo-workflows-1.0.jar).
```

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 4]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
...
+
+To install the extension:
+
+1. Add the business-networks-contracts dependency in your **contracts** (and states) cordapp module:
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:60]:
```suggestion
1. Add the `business-networks-contracts` dependency in your **contracts** (and states) CorDapp module:
```

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 5]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
...
+
+```
+dependencies {
+    //...
+    cordapp("net.corda.bn:business-networks-contracts:$corda_bn_extension_version")
+    //...
+}
+```
+2. Add the business-networks-workflows dependency in your "workflows" cordapp module:
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:69]:
```suggestion
2. Add the `business-networks-workflows` dependency in your **workflows** CorDapp module:
```

File: content/en/docs/corda-os/4.6/business-network-membership.md [hunk 6]
```diff
@@ -48,7 +48,37 @@ In a Business Network, you can assign different roles to members of the network.
...
+dependencies {
+    //...
+	cordapp("net.corda.bn:business-networks-workflows:$corda_bn_extension_version")
+    //...
+}
+```
+
+3. Add both dependencies in your **Cordform** - `deployNodes` - task.
+
+You have installed the business network membership extension.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/business-network-membership.md:81]:
```suggestion
You have installed the Business Network membership extension.
```

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]
A few suggestions added to the OS file - pls propagate to the CE file.

**Review by tubibuto — APPROVED** [review]
Looks good, let's wait also for @bpaunescu's sanity check.

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Review by bpaunescu — APPROVED** [review]
Thank you @benpesterr3 , this looks really good.

**Pull request merged**